general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -568,11 +597,12 @@ def aqt_get_configs(self) -> Dict[str, str]:
         """
         return self.get_request("/get_aqt_configs")
 
-    def get_request(self, endpoint: str) -> Any:
+    def get_request(self, endpoint: str, json_dict: Optional[Mapping[str, object]] = None) -> Any:
```

Comment:
we should use a `post_request` instead of adding a json body to a get request

Reply:
Done